examples/tutorials/data/hawc.py

```diff
@@ -289,6 +290,6 @@
 # ----------
 #
 # Now you know how to access and work with HAWC data. All other
-# tutorials and documentation concerning 3D analysis and `~gammapy.datasets.MapDataset`s
```

Comment:
So showing the complete path (not only `MapDatasets`) without the tilde and linking it to `gammapy.datasets.MapDataset`?

Reply:
yes... as made in the PR #5490...
I ahve a doubt: the link is towards `gammapy.datasets.MapDatasets` or `gammapy.datasets.MapDataset`?

Reply:
`gammapy.datasets.MapDatasets` does not exist. It was just a matter of language. Showing the plural but actually pointing to `gammapy.datasets.MapDataset`. Adding the 's':
```python
`gammapy.datasets.MapDataset`s
```
 at the end is making an incorrect rendering:
 
 
![immagine](https://github.com/user-attachments/assets/7afdbb4f-3ca0-4f6e-9ab2-36d208c142bf)

Reply:
oups... indeed. either MapDataset or Datasets! sorry Daniel

Reply:
Otherwise you could change the wording?

_All other tutorials and documentation concerning 3D analysis and the `~gammapy.datasets.MapDataset` object can be used from this step._

Reply:
I rephrased it like that.